client/blocks/post-share/publicize-actions-list.jsx

```diff
@@ -192,14 +189,11 @@ class PublicizeActionsList extends PureComponent {
 			);
 		}
 
-		if ( hasRepublicizeFeature && ! hasRepublicizeSchedulingFeature ) {
-			return <UpgradeToPremiumNudge { ...this.props } />;
-		}
 		if ( scheduledActions.length === 0 ) {
 			return <Notice
 				status="is-info"
 				showDismiss={ false }
-				text={ translate( 'Did you know you can decide exactly when Publicize shares your post? You can!' +
+				text={ translate( 'Did you know you can decide exactly when Publicize shares your post? You can! ' +
```

Comment:
Hi! I've found a possible matching string that has already been translated **1** time:
`translate( 'Did you know you can decide exactly when Publicize shares your post? You can!Click the calendar icon next to "Share post" to schedule your social shares.' )` *ES Score: 18*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).